app/apps/server/bridges/listeners.js

```diff
@@ -100,6 +102,16 @@ export class AppListenerBridge {
 					room: this.orch.getConverters().get('rooms').convertRoom(data.room),
 					agent: this.orch.getConverters().get('users').convertToApp(data.user),
 				});
+			case AppInterface.IPostLivechatRoomTransferred:
+				const converter = data.type === LivechatTransferEventType.AGENT ? 'users' : 'departments';
+
+				return this.orch.getManager().getListenerManager().executeListener(inte, {
+					type: data.type,
+					room: this.orch.getConverters().get('rooms').convertById(data.room),
+					from: this.orch.getConverters().get(converter).convertById(data.from),
```

Comment:
Since the `from` may not be assigned(like when a room was in the queue), do you think this may cause issues for the `converters`?

Sry, I just noticed that after revisiting here.

Reply:
No, in the case when the value the converter receives is `undefined` it will just return `undefined` as well :)